config/features.yml

```diff
@@ -333,9 +333,13 @@ features:
     actor_type: user
     description: When enabled, claims status tool uses DataDog's Real User Monitoring logging
     enable_in_development: false
-  cst_suppress_evidence_requests:
+  cst_suppress_evidence_requests_website:
```

Comment:
Since this change will delete one flipper and create a new one, you'll need to delete `cst_suppress_evidence_requests`. These are the instructions for that:

Once this is deployed, you'll need to follow step 3 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.

Reply:
Thank you! I was coincidentally looking at this the other day.